### ISSUE TITLE
Exception in Polaris 9.2.0 when PDB uses spec.maxUnavailable instead of spec.minAvailable

### ISSUE THREAD

**Issue by dnwlf** [FairwindsOps/polaris#1061]
### What happened?

After upgrading to Polaris 9.2.0, we get the attached exception in our pipelines:
[pipeline-output.txt](https://github.com/user-attachments/files/16285206/pipeline-output.txt)

Some of our PodDisruptionBudgets are configured to use `spec.maxUnavailable` rather than `spec.minAvailable`, which is valid according to the `PodDisruptionBudget` spec, and which is even included in [Kubernetes documentation examples](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget).

I believe the root problem is in the `pdbMinAvailableGreaterThanHPAMinReplicas` function, which does not check to see if `attachedPDB.Spec.MinAvailable` has a value before passing it to `getIntOrPercentValueSafely`, in [this section of the function](https://github.com/FairwindsOps/polaris/commit/952b6aed65ed971bf64a64c2698c3df3905248cc#diff-8cf7e477f7fe55b5d41511f169b90a87607ea711e4fd7c894d77049aa020fde9R50-R54)

### What did you expect to happen?

I expect Polaris to not throw an exception for PDBs that adhere to the PodDisruptionBudget spec.

### How can we reproduce this?

1. Save the following as `app.yaml`
    ```yaml
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:  
      name: "app-pdb"
      labels:
        app.kubernetes.io/name: "app"
    spec:
      maxUnavailable: 25%
      selector:
        matchLabels:
          app.kubernetes.io/name: "app"
    ---
    apiVersion: apps/v1
    kind: Deployment
    metadata:  
      name: "app-deployment"
      labels:
        app.kubernetes.io/name: "app"
    spec:  
      selector:
        matchLabels:
          app.kubernetes.io/name: "app"
      revisionHistoryLimit: 3
      template:
        metadata:
          labels:
            app.kubernetes.io/name: "app"
        spec:
          containers:
            - name: container
              image: "alpine"
              resources: {}
    ---
    apiVersion: autoscaling/v2
    kind: HorizontalPodAutoscaler
    metadata: 
      name: "app-hpa"
      labels:
        app.kubernetes.io/name: "app"
    spec:
      scaleTargetRef:
        apiVersion: apps/v1
        kind: Deployment
        name: app-deployment
      minReplicas: 2
      maxReplicas: 4
      metrics:
      - type: Resource
        resource:
          name: cpu
          target:
            type: Utilization
            averageUtilization: 70
    ```
 1. using Polaris version 9.2.0, run `polaris audit --audit-path app.yaml`

### Version

9.2.0

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sudermanjr** [comment]
Thanks for the report! Fix PR is in if you want to take a look. Should be released sometime soon.

**Comment by dnwlf** [comment]
Thank you so much for the incredibly fast response and fix!

**Comment by vitorvezani** [comment]
Released on `9.2.1` 🥳
